src/test/java/de/berlin/lostberlin/service/BusinessServiceTest.java

```diff
@@ -29,30 +43,30 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
-/*
-Unfortunately, I get the same problem with @JpaAuditing annotation even after it's been externalized to ApplicationConfig file. So I had to comment it out for making tests.
-I added @ContextConfiguration because ApplicationContext could not be loaded without it.
- */
 
 @RunWith(SpringRunner.class)
-@ContextConfiguration(classes = ApplicationConfig.class, initializers = ConfigFileApplicationContextInitializer.class)
-@SpringBootTest
+@DataJpaTest
+@AutoConfigureTestDatabase(replace= AutoConfigureTestDatabase.Replace.NONE)
 public class BusinessServiceTest {
 
-    @MockBean
-    private BusinessRepository businessRepository;
+    @Autowired
+    BusinessService businessService;
 
-    private BusinessService businessService;
-
-    @MockBean
-    ValidatorFactory factory;
+    @TestConfiguration
+    @EnableJpaAuditing
```

Comment:
We use an inner class with special annotation `@TestConfiguration` in order to create a test bean of our BusinessService.
We `@EnableJpaAuditing` for test beans in order to deal with createdAt and modifiedAt auditing in our test service.